gemfiles/rails_5.gemfile

```diff
@@ -2,6 +2,7 @@
 
 source "https://rubygems.org"
 
+gem "sqlite3", "< 1.4.0"
```

Comment:
Why would we need to lock sqlite everywhere?

Reply:
Hey @pkuczynski 
After release of sqlite3 gem version 1.4.0, tests failed because of 
```LoadError:
  Error loading the 'sqlite3' Active Record adapter. Missing a gem it depends on? can't activate sqlite3 (~> 1.3.6), already activated sqlite3-1.4.1. Make sure all dependencies are added to Gemfile.
```
It is happening because rails requires it [here|https://github.com/rails/rails/blob/5-2-stable/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb#L12]

Reply:
But that should happen only for newer version of rails, right?

Reply:
If so, we could do more logic on the `config.gemspec` as we do in some other cases.

Reply:
No, it works the same for every version of Rails (it's because rails formally does not depend on any db adapter, that is why it requires adapter gem at the runtime)

Reply:
Well, how did it worked before then? :)